examples/existing_vpc/output.tf

```diff
@@ -0,0 +1,13 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "vpc_id" {
+  value       = module.slz_vpc.vpc_id
+  description = "VPC id"
+}
+
+output "vpc_crn" {
+  value       = module.slz_vpc.vpc_crn
+  description = "VPC crn"
+}
```

Comment:
In order to ensure we have the outputs correct for this use case, can you output everything that the root level module outputs so we can check all values in the test. Thanks